openlibrary/templates/type/edition/view.html

```diff
@@ -157,14 +157,17 @@
 
 $if is_privileged_user:
   $:render_template("type/edition/admin_bar", work, edition, ocaid)
+$ render_summary = None
+$ render_summary= render_template("type/edition/render_summary",page,work,edition,ocaid,work_title)
+
```

Comment:
```suggestion

$ component_times['ReadlogStats'] = time()
$ star_ratings_stats = macros.StarRatingsStats(work)
$ component_times['ReadlogStats'] = time() - component_times['ReadlogStats']

$ render_summary= render_template("type/edition/render_summary", work, edition, ocaid, work_title, star_ratings_stats)
```
We can render the star rating stats component here.

By the way, we use `component_times` to measure how long components take to render while the book page is loading.  You can see these times by adding `?debug=true` to the end of a book page URL.